Stateless/StateConfiguration.cs

```diff
@@ -372,11 +424,12 @@ public StateConfiguration PermitDynamic<TArg0>(TriggerWithParameters<TArg0> trig
             /// that the trigger will cause a transition to.</param>
             /// <param name="guard">Function that must return true in order for the
             /// trigger to be accepted.</param>
+            /// <param name="guardDescription">Guard description</param>
             /// <returns>The reciever.</returns>
-            public StateConfiguration PermitDynamicIf(TTrigger trigger, Func<TState> destinationStateSelector, Func<bool> guard)
+            public StateConfiguration PermitDynamicIf(TTrigger trigger, Func<TState> destinationStateSelector, Func<bool> guard, string guardDescription = "")

```

Comment:
Might have missed a couple of `= null` defaults; this one and the few below use the empty string, which won't result in the method name being used as a default. Cheers!